src/dom/dom.js

```diff
@@ -713,6 +713,18 @@ p5.prototype.createSelect = function() {
       }
     }
   };
+
+  self.disable = function(value) {
```

Comment:
can you add documentation for this method?

Reply:
Working on it.

Reply:
Added documentation in commit 4136c5c